library/Icinga/Web/Cookie.php

```diff
@@ -262,4 +269,13 @@ public function setValue($value)
         $this->value = $value;
         return $this;
     }
+
+    public function forgetMe()
+    {
+        $this->object = $this;
```

Comment:
Cloning works different. You are just assigning the pointer to the instance to a new variable. With `$this->object` you still refer to the same instance in `$this`. Please see https://www.php.net/manual/en/language.oop5.references.php and https://www.php.net/manual/en/language.references.php. In general this is "pass by value" and "pass by reference".  For cloning please see https://www.php.net/manual/en/language.oop5.cloning.php.